doc/source/examples/03_modeling/scale_map_bodies.mystnb

```diff
@@ -0,0 +1,233 @@
+   - Copy the original cube. By using `scale` with a value of 0.5, we will effectively halve the side lengths of the cube, and then offset it to view the difference.
+   - Note: Since we doubled the size of the cube in the previous cell, we need to use the 0.25 factor which in turn will translate to half the size of the original cube.
+
+```{code-cell} ipython3
+# Copy the original cube
+halved = cube.copy(cube.parent_component, "halved_box")
+# Half the size
+halved.scale(0.5)
+# Translate the copied cube in the x direction
+halved.translate(UnitVector3D([1, 0, 0]), -25)
+
+design.plot()
+```
+
+## Map Function
+
+The `map` function enables the reorientation of 3D bodies by mapping them onto a new specified frame. This function is used for adjusting the orientation of geometric bodies within 3D space to match specific reference frames. Through this, the user is able to effectively perform translation and rotation operations in a single method by specifying a new frame.
+
+### Usage
+
+To use the `map` function, invoke it on an instance of a geometry body with a single argument: the new frame to which the body should be mapped. The frame is a structure or object that defines the new orientation parameters for the body.
+
+```python
+body.map(new_frame)
+
++++
+
+## Example: Creating a Asymmetric Cube
+The code snippets below demonstrates how to reframe a cube body using `map` function in `body.py`. The process involves initializing a sketch design for the custom body, extruding the profile by a distance, and then performing the mapping operation to rotate the shape.
+1. **Initialize the Shape Sketch Design:**
+   - A new design sketch named "asymmetric_cube" is created.
+
+```{code-cell} ipython3
+# Initialize the sketch design
+design = m.create_design("asymmetric_cube")
+```
+
+2. **Create Asymmetric Sketch Profile:**
+   - We will make a sketch profile which is basically a cube centered on the origin with side length 2 with a cutout.
+
+```{code-cell} ipython3
+# Create the cube profile with a cut through it
+asymmetric_profile = Sketch().segment(Point2D([1, 1]), Point2D([-1, 1])).segment_to_point(Point2D(
+    [0, 0.5])).segment_to_point(Point2D([-1, -1])).segment_to_point(Point2D([1, -1])).segment_to_point(Point2D([1, 1]))
+```
+
+3. **Create Asymmetric Body:**
+   - `extrude_sketch` on the `asymmetric_profile`, as the base sketch, creating the 3D cube, with a cutout, with `distance` being 1.
+
+```{code-cell} ipython3
+# Extrude the asymmetric profile by distance of 1 unit
+body = design.extrude_sketch("box", asymmetric_profile, 1)
+
+design.plot()
+```
+
+4. **Apply Map Reframing:**
+   - We will first make a copy of the shape and translate it in 3D space so that we can view them side by side, then we will apply the reframing to the copied shape.
+   - Note: The below map uses the default x direction, but the y direction is swapped with the z direction, effectively rotating the original shape so that it is standing vertically.
```

Comment:
```suggestion
   - First make a copy of the shape and translate it in 3D space so that you can view them side by side. Then, apply the reframing to the copied shape.
   - Note: The following map uses the default x direction, but the y direction is swapped with the z direction, effectively rotating the original shape so that it is standing vertically.
```